config/config.go

```diff
@@ -1510,13 +1450,8 @@ func GetFilePath(configFile string) (configPath string, isImplicitDefaultPath bo
 // config directory as `File` or `EncryptedFile` depending on whether the config
 // is encrypted
```

Comment:
Out of scope: But while you are here can you change the log.Warn to fmt.Printlns for SetNTPCheck; as they don't appear if log levels are not set and the code block pauses execution. 
![image](https://github.com/user-attachments/assets/f53af2ac-7336-4fcf-948d-e89a39c84a7d)

Reply:
Yeah, since this bug isn't too important if it's lost, I can driveby it now.
I'm unhappy with the theme  of user-interaction from the CLI, where we act more like a windows app hand-holding the user with pop-up prompts on edge cases instead of just bailing on the user and telling them to fix it.
We have a lot of code, maintenance, bugs, dedicated to user interaction for paths that are one-off, rare, and could be error and bail, and I think this is one of them. That said: Backing away...

Fixed 95cd6c21b